docs/source/cookbook.rst

```diff
@@ -199,5 +199,11 @@ FIXME
 Using a Different GPG Base Directory
 ====================================
 
-FIXME
+PyGPGME honours the ``GNUPGHOME`` environment variable. This means you can change the GPG Base Directory simply by setting it using ``os.environ``::
+
+    import os
+    import gpgme
+
+    os.environ['GNUPGHOME'] = '/path/to/dir'
+    ctx = gpgme.Context()
```

Comment:
Seeing as you've come up with a LOT better method, I'd be happy to abandon the above PR and let you create the PR with the above details.